apps/web/src/pages/submission/[step].tsx

```diff
@@ -9,34 +10,70 @@ const Submission = () => {
 
   const step = Number(router.query.step);
 
+  // do not scroll to the top of the form on initial load
+  const formRef = useRef<HTMLDivElement>(null);
+  const prevStepRef = useRef<number>(1);
+  useEffect(() => {
+    if (!isNaN(step) && prevStepRef.current !== step && formRef.current) {
+      window.scrollTo({ left: 0, top: formRef.current.offsetTop, behavior: 'smooth' });
+      prevStepRef.current = step;
+    }
+  }, [step]);
+
   return (
     <>
-      <h1 className='text-4xl mb-3'>Health Provider Registry for BC’s Emergency Response</h1>
-      <section className='mb-4'>
-        <p>
-          Please refer to the <ExternalLink href='#'>FAQ’s</ExternalLink> when completing this form.
+      <h1 className='text-3xl mb-7 mx-6 lg:text-4xl lg:mb-3 lg:mx-0'>
+        Health Provider Registry for BC’s Emergency Response
+      </h1>
+      <section className='mb-7'>
+        <p className='mb-4'>
+          The <b>Emergency Health Provider Registry (EHPR)</b> is an online registry to support the
+          proactive deployment of health care providers to ensure BC’s health care system is best
+          prepared to respond to emergencies (e.g., wildfires, floods, pandemics). It is an online
+          registry of health care professionals and health authority staff who are able and willing
+          to be deployed or hired to support B.C.’s health system response. For more information
+          about the EHPR, please refer to the&nbsp;
+          <ExternalLink href='https://www.heu.org/sites/default/files/2021-08/FAQ%20EHPR_12August2020.pdf'>
+            FAQs
+          </ExternalLink>
+          .
+        </p>
+        <p className='mb-4'>
+          All&nbsp;
+          <ExternalLink href='https://www.heu.org/sites/default/files/2021-08/FAQ%20EHPR_12August2020.pdf'>
+            eligible health care providers or health care staff
+          </ExternalLink>
+          &nbsp;are invited to register using the form below. Health authorities, the Ministry of
+          Health or HealthMatch BC may use the EHPR to initiate contact if/when assistance is
+          required.
         </p>
-        <p>
-          If you encounter problems completing/submitting this form, please email{' '}
+        <p className='mb-4'>
+          The form takes about 10 minutes to complete - if you are in a career stream with oversight
+          from a regulatory or credentialing body (e.g., BC College of Nursing Professionals),
+          please have your registration number available.
+        </p>
+        <p className='mb-4'>
+          <b>If you encounter problems completing/submitting this form, please email </b>
```

Comment:
> If you encounter problems completing this form, please email EHPRQuestions@gov.bc.ca.

The text from the doc only says `completing` rather than `completing/submitting`, do we have different sources for copy?

Reply:
My mistake.